lib/util.js

```diff
@@ -662,10 +662,70 @@ const CLEANUP_DEFAULTS = {
             'border', 'crossorigin', 'height', 'hspace', 'ismap', 'longdesc', 'sizes', 'src', 'srcset',
             'usemap', 'vspace', 'width',
         ],
+        'a': ['target'],
     },
     noComments: true,
 };
 
+function _mergeObjectArrays(obj1, obj2) {
+    const keys1 = new Set(Object.keys(obj1));
+    const keys2 = new Set(Object.keys(obj2));
+    const toMerge = new Set([...keys1].filter(key => keys2.has(key)));
+
+    const merged = {};
+    toMerge.forEach(key => {
+        merged[key] = [
+            ...obj1[key],
+            ...obj2[key],
+        ];
+    });
+
+    return Object.assign(obj1, obj2, merged);
+}
+
+/**
+ * Utility to extend {@link utilcleanup_body} defaults.
+ *
+ * @param {Object} options - Pass an Object with the same format as
+ *   {@link utilCLEANUP_DEFAULTS}.
+ * @returns {Object} - This is a merge of the {@link
+ *   utilCLEANUP_DEFAULTS} and the passed options.
+ * @memberof util
+ */
+function extend_cleanup_defaults({
```

Comment:
What about `extend_cleanup_body_defaults` to keep consistency.

Reply:
Hmm I tried and started to port ingesters, but is too long. The variable with defautls is called `CLEANUP_DEFAULTS` so I think prepending `extend_...` to it is OK.